internal/build/phase_test.go

```diff
@@ -133,7 +133,15 @@ func testPhase(t *testing.T, when spec.G, it spec.S) {
 					phaseName,
 					lifecycle,
 					build.WithArgs("read", "/workspace/fake-app-file"),
-					build.WithContainerOperations(lifecycle.CopyApp),
+					build.WithContainerOperations(
+						build.CopyDir(
```

Comment:
I would love to see us call `build.WriteStackToml` here - is it possible? That said, with the introduction of internal/build/container_ops_test.go, perhaps it's not necessary.